src/cloudant/design_document.py

```diff
@@ -225,6 +333,15 @@ def list_views(self):
         """
         return list(self.views.keys())
 
+    def list_indexes(self):
+        """
+        Retrieves a list of available search indexes in the locally
+        cached DesignDocument indexes dictionary.
+
+        :returns: List of search index names
```

Comment:
This doc block has refers to "search indexes" and returns "search index" names. I think the word search should be removed from both these places since it will list any indexes right?